api/src/main/java/org/openmrs/module/fgh/mpi/MpiHttpClient.java

```diff
@@ -195,11 +357,11 @@ private <T> T submitRequest(String requestPath, String data, Class<T> responseTy
 	private void initIfNecessary() throws Exception {
 		
 		synchronized (this) {
-			//Add global property listener to rebuild the SSL context when the GP values change
+			//Add global property listener to rebuild the SSL contexte when the GP values change
```

Comment:
I think this is a stray change, please undo it

Reply:
You are right